projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/base/LLVMContext.java

```diff
@@ -102,4 +104,16 @@ public Source getSourceFile() {
         return sourceFile;
     }
 
+    // TODO No static access to this class from LLVMFunction at the moment
+
+    private static LLVMContext lastContext;
+
+    public static CallTarget getCallTarget(LLVMFunction function) {
+        return lastContext.registry.lookup(function);
+    }
+
+    public static LLVMStack getStaticStack() {
+        return lastContext.stack;
+    }
+
```

Comment:
`LLVMContext` is not accessible from `LLVMFunction` - see code there for why we need to do it.